texar/torch/modules/classifiers/bert_classifier.py

```diff
@@ -71,6 +71,8 @@ def __init__(self,
 
         super().__init__(hparams=hparams)
 
+        self.load_pretrained_config(pretrained_model_name, cache_dir)
```

Comment:
If I understand it correctly, will `load_pretrained_config` and `init_pretrained_weights` be called twice (once in `BERTClassifier`, and once in `BERTEncoder`)?

Reply:
If that is the case, we probably should not load the pre-trained weights in `self.encoder (BERTEncoder)`.

Reply:
Discussed offline.
We can pass `pretrained_model_name` as None while instantiating the encoder in `BERTClassifier`.

Reply:
If you set `pretrained_model_name` and `pretrained_model_name` in `hparams` to be `None`, `BERTEncoder` won't load the pre-trained weights.

Reply:
Made both the changes.